src/style.css

```
@@ -1 +1,78 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.content {
+  font-family: "Roboto" , sans-serif;
+  font-style: normal;
+}
```

better to specify this style for the whole HTML 
```suggestion
html {
  font-family: "Roboto" , sans-serif;
  font-style: normal;
}
```